Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,33 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
```

Comment:
I think don't think `onclick` attribute is necessary, since your javascript is already catching it.